### ISSUE TITLE
Μοντελο Προπαρασκευης

### ISSUE THREAD

**Issue by mpektkd** [slp-ntua/slp-labs#66]
Καλησπερα, θα ηθελα να κανω μια ερωτηση για το DNN της προπαρασκευης. Πειραζει, αν για το binary classification χρησιμοποιησουμε το crossentropy και οχι το BGE, καθως δεν ειμαι πολυ σιγουρος για το πως να το χρησιμοποιησω ειχα κανει μια προσπαθεια αλλα δεν μου βγαζε καθολου καλα αποτελεσμα και δοκιμασα με το crossentropy και για το trainset βγαζω accuracy κοντα στη μοναδα και για το testset 80%.

**Comment by pxaris** [comment]
Καλημέρα, καταλαβαίνω ότι το νευρωνικό που χρησιμοποιείς έχει 2 εξόδους (νευρώνες στο τελευταίο layer). Στην περίπτωση αυτή, μπορείς να χρησιμοποιήσεις το `CrossEntropyLoss` αντί του `BCEWithLogitsLoss`. Μια σχετική συζήτηση μπορεί να βρεθεί [εδώ](https://stackoverflow.com/questions/53628622/loss-function-its-inputs-for-binary-classification-pytorch).

**Comment by mpektkd** [comment]
Ναι ακριβως αυτο γινεται. Ωραια σας ευχαριστω πολυ.
